css/styles.css

```diff
@@ -0,0 +1,21 @@
+@import url("footer.css");
+@import url("card.css");
+@import url("form.css");
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+h1 {
+  text-align: center;
+  margin: 0;
+  color: whitesmoke;
+}
+.header {
+  background-color: cadetblue;
+  margin: 0;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  position: sticky;
+}
```

Comment:
this should be outsourced into a `globals.css` file. The index should only contain imports.